src/material/dialog/testing/dialog-harness.ts

```diff
@@ -11,56 +11,57 @@ import {DialogRole} from '@angular/material/dialog';
 import {DialogHarnessFilters} from './dialog-harness-filters';
 
 /**
- * Harness for interacting with a standard MatDialog in tests.
+ * Harness for interacting with a standard `MatDialog` in tests.
  * @dynamic
  */
 export class MatDialogHarness extends ComponentHarness {
   // Developers can provide a custom component or template for the
   // dialog. The canonical dialog parent is the "MatDialogContainer".
+  /** The selector for the host element of a `MatDialog` instance. */
   static hostSelector = '.mat-dialog-container';
 
   /**
-   * Gets a `HarnessPredicate` that can be used to search for a dialog with
-   * specific attributes.
-   * @param options Options for narrowing the search:
-   *   - `id` finds a dialog with specific id.
+   * Gets a `HarnessPredicate` that can be used to search for a `MatDialogHarness` that meets
+   * certain criteria.
+   * @param options Options for filtering which dialog instances are considered a match.
    * @return a `HarnessPredicate` configured with the given options.
    */
   static with(options: DialogHarnessFilters = {}): HarnessPredicate<MatDialogHarness> {
     return new HarnessPredicate(MatDialogHarness, options);
   }
 
-  /** Gets the id of the dialog. */
+  /** Gets a promise for the id of the dialog. */
   async getId(): Promise<string|null> {
     const id = await (await this.host()).getAttribute('id');
     // In case no id has been specified, the "id" property always returns
     // an empty string. To make this method more explicit, we return null.
     return id !== '' ? id : null;
   }
 
-  /** Gets the role of the dialog. */
+  /** Gets a promise for the role of the dialog. */
   async getRole(): Promise<DialogRole|null> {
     return (await this.host()).getAttribute('role') as Promise<DialogRole|null>;
   }
 
-  /** Gets the value of the dialog's "aria-label" attribute. */
+  /** Gets a promise for the value of the dialog's "aria-label" attribute. */
   async getAriaLabel(): Promise<string|null> {
     return (await this.host()).getAttribute('aria-label');
   }
 
-  /** Gets the value of the dialog's "aria-labelledby" attribute. */
+  /** Gets a promise for the value of the dialog's "aria-labelledby" attribute. */
   async getAriaLabelledby(): Promise<string|null> {
     return (await this.host()).getAttribute('aria-labelledby');
   }
 
-  /** Gets the value of the dialog's "aria-describedby" attribute. */
+  /** Gets a promise for the value of the dialog's "aria-describedby" attribute. */
   async getAriaDescribedby(): Promise<string|null> {
     return (await this.host()).getAttribute('aria-describedby');
   }
 
   /**
-   * Closes the dialog by pressing escape. Note that this method cannot
-   * be used if "disableClose" has been set to true for the dialog.
+   * Closes the dialog by pressing escape.
+   *
+   * Note: this method cannot be used if `disableClose` has been set to `true` for the dialog.
```

Comment:
This says "cannot be used", but I would instead say something like "Does nothing if the `disableClose` option has been set for the dialog"